packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -399,25 +407,29 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
 				/>
 			) }
 			<div className={ 'design-picker__grid' }>
-				{ filteredDesigns.map( ( design ) => (
-					<DesignButtonContainer
-						key={ design.slug }
-						design={ design }
-						locale={ locale }
-						onSelect={ onSelect }
-						onPreview={ onPreview }
-						onUpgrade={ onUpgrade }
-						highRes={ false }
-						hideFullScreenPreview={ false }
-						hideDesignTitle={ false }
-						isPremiumThemeAvailable={ isPremiumThemeAvailable }
-						previewOnly={ previewOnly }
-						hasDesignOptionHeader={ hasDesignOptionHeader }
-						onCheckout={ onCheckout }
-						verticalId={ verticalId }
-						hasPurchasedTheme={ wasThemePurchased( purchasedThemes, design ) }
-					/>
-				) ) }
+				{ filteredDesigns.map( ( design ) =>
+					! isBlankCanvasDesign( design ) ? (
```

Comment:
We already have a bunch of `isBlankCanvasDesign()` logic already inside the `DesignButtonContainer` component. So perhaps we can render the `PatternAssemblerCta` there as well, to avoid duplication here?

Reply:
Good idea